app/internal/page/certificateprovider/your_details.go

```diff
@@ -0,0 +1,128 @@
+package certificateprovider
+
+import (
+	"net/http"
+	"strings"
+
+	"github.com/ministryofjustice/opg-modernising-lpa/internal/page"
+	"github.com/ministryofjustice/opg-modernising-lpa/internal/sesh"
+
+	"github.com/gorilla/sessions"
+	"github.com/ministryofjustice/opg-go-common/template"
+	"github.com/ministryofjustice/opg-modernising-lpa/internal/date"
+	"github.com/ministryofjustice/opg-modernising-lpa/internal/validation"
+)
+
+type cpYourDetailsData struct {
```

Comment:
since these are now in a separate package we can call them `yourDetails...` and not have conflicts with the donor stuff

Reply:
Missed these - will update.